references/cuegen/convert.go

```diff
@@ -77,7 +77,7 @@ func (g *Generator) convertDecls(x *goast.GenDecl) (decls []cueast.Decl, _ error
 
 func (g *Generator) convert(typ gotypes.Type) (cueast.Expr, error) {
 	// if type is registered as any, return {...}
-	if _, ok := g.anyTypes[typ.String()]; ok {
+	if _, ok := g.opts.anyTypes[typ.String()]; ok {
```

Comment:
nil check?

Reply:
`Generate` ensures that `opts!=nil`, otherwise `convert.go` might be filled with `if opts!=nil`...

Reply:
Updated